diesel_tests/tests/schema.rs

```diff
@@ -57,33 +57,11 @@ pub struct Comment {
     text: String,
 }
 
-// Compiler plugin will automatically invoke this based on schema
-table! {
-    users {
-        id -> Serial,
-        name -> VarChar,
-        hair_color -> Nullable<VarChar>,
-    }
-}
+load_table_from_schema!(dotenv!("DATABASE_URL_FOR_SCHEMA"), "users");
```

Comment:
I wonder if `infer_table_from_schema` is a better name here. `load` implies that we're doing `SELECT * FROM` to me.